Assets/Engine/Tile/TileRework/TileManager.cs

```diff
@@ -301,7 +302,7 @@ public bool CanBuild(TileMap selectedMap, int subLayerIndex, TileObjectSO tileOb
         /// <param name="position"></param>
         /// <param name="dir"></param>
         /// <returns></returns>
-        public bool CanBuild(TileObjectSO tileObjectSO, Vector3 position, Direction dir)
+        public bool CanBuild(TileObjectSo tileObjectSO, Vector3 position, Direction dir)
```

Comment:
Not sure if you wanted to rename any of these other "SO"s to "So" or if you intentionally avoided these.
![image](https://user-images.githubusercontent.com/41941008/147693644-a4c39ac5-8883-458f-99e6-26da97ffa9f6.png)

Reply:
Yup, that was deliberate, I just renamed the class. Renaming all variables added too many file changes that were unrelated to this PR. I'll use a linter at some point and clean up all these violations automatically within some other PR.